filer/templates/admin/filer/widgets/admin_file.html

```diff
@@ -11,24 +11,25 @@
     </div>
 
     <div class="js-dropzone dropzone" data-url="{% url 'admin:filer-ajax_upload' %}">
-        <div class="dz-default dz-message js-dropzone-message{% if object %} hidden{% endif %}" id="dropzone-message">
+        <div class="dz-default dz-message js-dropzone-message{% if object %} hidden{% endif %}" id="{{ id }}_dropzone_message">
```

Comment:
id identifiers are not allowed to start with a number (yet)

i'm sure it's not really a problem for our purposes but i'd still avoid